### PR TITLE
Add note to eligibility if widowed

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- To make clear that the following answers (in case you're widowed since last year) are made for both parties, we add an explaining text

# Changes
- Add block to the standard eligibility form for additional info
- Use different template for the marital_status step
- Add additional note to that template
- Conditionally show and hide the note

# Feedback
- Is this what you expected?
- Do you have any idea why this also removes the `"form.eligibility.marital_status-label"` messages and such? (see bottom of messages.po)

### CODE REVIEW

File: webapp/app/templates/eligibility/form_marital_status_input.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <script>
+        function show_hide_widowed_note() {
+            if ($('input[value="widowed"]').is(':checked')) {
+                $('#marital_status_widowed_note').show();
+            } else {
+                $('#marital_status_widowed_note').hide();
+            }
+        }
+
+        $(`input[type="radio"]`).change(show_hide_widowed_note)
```
Comment (JulianRoesner) [inline webapp/app/templates/eligibility/form_marital_status_input.html:21]: Why not this?
```suggestion
        $(`input[value="widowed"]`).change(show_hide_widowed_note)
```

File: webapp/app/templates/eligibility/form_full_width.html [hunk 2]
```diff
@@ -10,4 +10,6 @@
     {% for field in form %}
         {{ macros.render_field(field, 12, position_details_after=True) }}
     {% endfor %}
+    {% block additional_form_content %}
+    {% endblock %}
```
Comment (JulianRoesner) [inline webapp/app/templates/eligibility/form_full_width.html:14]: You do not need this. Just use `block form_content` with super in form_marital_status_input.html

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -4796,51 +4726,59 @@ msgstr ""
...
 msgid "form.eligibility.use-register-button"
 msgstr "Registrieren"
 
+#: app/templates/eligibility/form_marital_status_input.html:6
+msgid "eligibility.marital_status.widowed_note"
+msgstr ""
+"Wenn Sie seit 2020 verwitwet sind und zusammen gelebt haben, bietet Ihnen"
+" der Steuerlotse nur die Möglichkeit, eine gemeinsame Steuererklärung für"
+" Sie beide zu machen (Zusammenveranlagung). Ihre Antworten müssen sich im"
+" Folgenden daher auf Sie beide beziehen."
```
Comment (JulianRoesner) [inline webapp/app/translations/de/LC_MESSAGES/messages.po:4780]: I know this is not the correct place. However, I am wondering if I am the only one for whom the text "Sie beide" seems a little strange.

**Review by JulianRoesner — APPROVED** [review]
I've expected something different. However, I did not really understand the additional note correctly. I think we'll have to look again into this when we have our new frontend architecture in place. Until then it looks good to me.

The labels were deleted in an older commit. I guess, that `marital_status-label` has just been overlooked.

File: webapp/app/translations/de/LC_MESSAGES/messages.po (hunk 3 above)
Comment (frederike-ramin) [reply webapp/app/translations/de/LC_MESSAGES/messages.po:4780]: Haha, I personally don't ind it but I see your point. :D

File: webapp/app/templates/eligibility/form_marital_status_input.html (hunk 1 above)
Comment (frederike-ramin) [reply webapp/app/templates/eligibility/form_marital_status_input.html:21]: I also thought that would work. BUT: this way, the text ist not hidden when another radio button is clicked. :)

**Pull request merged**